ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml

```diff
@@ -314,6 +314,38 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-aws-serial
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le
+  cron: 20 11,23 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: ppc64le
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      GO_VERSION: 1.20.7
+      JQ_VERSION: "1.6"
+      LOKI_ENABLED: "false"
+      OCP_VERSION: "4.14"
+      RESOURCE_GROUP: ipi-cicd-resource-group
+      TERRAFORM_VERSION: 1.5.5
+      TEST_SKIPS: deploymentconfigs\| should expose cluster services outside the cluster\|
```

Comment:
This may already be available, but we should probably work with Shilpa to keep a catalogue of the tests we've disabled and why we've disabled them.

That way we can review the skip list on a regular cadence and unskip them as we deal with root causes.